tools/install.pl

```diff
@@ -127,12 +127,14 @@
     # libarchive is not provided by default on macOS, so we set the correct env vars
     # to successfully compile Archive::Extract::Libarchive and Archive::Peek::Libarchive
     my $pre = "";
+    my $suff = "";
     if ($Config{"osname"} eq "darwin") {
```

Comment:
Since `$suff` is necessary for Homebrew-based installations to work, putting this under a `darwin` condition will still break Linuxbrew.  

I guess it should be put under a separate `if $SENV{HOMEBREW_FORMULA_PREFIX}` ?

Reply:
Good point. I changed it!